services/QuillLMS/app/helpers/quill_academy_helper.rb

```diff
@@ -0,0 +1,105 @@
+    },
+    QUILLS_INDEPENDENT_TOOLS => {
+      header: QUILLS_INDEPENDENT_TOOLS,
+      text: "Learn more about our independent activity grammar tools that unlock the power of feedback."
+    },
+    USING_QUILL_LESSONS => {
+      header: USING_QUILL_LESSONS,
+      text: "Learn more about Quill Lessons which is our interactive writing tool that helps teachers lead students through live grammar lessons. In this course, you will learn how to use and implement the tool to maximize group learning."
+    },
+    USING_QUILL_READING_FOR_EVIDENCE => {
+      header: USING_QUILL_READING_FOR_EVIDENCE,
+      text: "Learn more about Quill Reading for Evidence which provides students with nonfiction texts paired with AI-powered writing prompts, instead of multiple-choice questions, to enable deeper thinking."
+    },
+    UNDERSTANDING_QUILL_DATA_REPORTS => {
+      header: UNDERSTANDING_QUILL_DATA_REPORTS,
+      text: "Quill offers a range of student data reports to help you make sense of student performance. In this course, you will learn how to navigate these reports."
+    },
+    DATA_AND_INSTRUCTION_BEST_PRACTICES => {
+      header: DATA_AND_INSTRUCTION_BEST_PRACTICES,
+      text: "Explore strategies to intentionally embed Quill into your instructional plans and the data reports that can provide immediate insight into student learning."
+    },
+    EXPLORING_THE_QUILL_ACTIVITY_LIBRARY => {
+      header: EXPLORING_THE_QUILL_ACTIVITY_LIBRARY,
+      text: "Explore the different pathways to assigning the just-right content to your students, including pre-made activity packs and curating your own packs using the Quill activity library."
+    },
+    IMPROVING_SENTENCE_FLUENCY => {
+      header: IMPROVING_SENTENCE_FLUENCY,
+      text: "Explore ways to expand students' work on the Quill platform to improve their sentence fluency in their writing beyond Quill."
+    },
+    ENCOURAGING_AND_EMPOWERING_YOUR_WRITERS => {
+      header: ENCOURAGING_AND_EMPOWERING_YOUR_WRITERS,
+      text: "Explore ways to encourage and increase students' writing self-efficacy using Quill's tools beyond extrinsic carrot-and-stick approaches."
+    },
+    USING_QUILL_TO_SUPPORT_STUDENTS_EXCEPTIONAL_LEARNERS => {
+      header: USING_QUILL_TO_SUPPORT_STUDENTS_EXCEPTIONAL_LEARNERS,
+      text: "Learn how to modify Quill's learning plans for your students with IEPs, provide targeted support, and track students' growth and progress."
+    },
+    USING_QUILL_TO_SUPPORT_ENGLISH_LANGUAGE_LEARNERS => {
+      header: USING_QUILL_TO_SUPPORT_ENGLISH_LANGUAGE_LEARNERS,
+      text: "Explore Quill's content created specifically for English Language Learners and learn strategies to utilize these resources to support students' written language acquisition."
+    }
+  }
+
+  def should_render_quill_academy_button
```

Comment:
I would get rid of the 'should' prefix since it makes it look like a spec method.  Also, i would append a '?' since it's a boolean method:  `render_quill_academy_button?`

Reply:
makes sense, I'll make those changes!